tasks/5_profile-lookup/profile-lookup.js

```diff
@@ -0,0 +1,43 @@
+export function lookupProfile(firstName, prop) {
+  let foundContact = contacts.find((person) => {
+    return person.firstName === firstName;
+  }
```

Comment:
Make it in the one line

Reply:
I didn't make it in one line, cause then the line exceeds maximum length
![Screen Shot 2019-09-10 at 6 33 55 PM](https://user-images.githubusercontent.com/33729448/64623215-a4665c80-d3f9-11e9-99ac-883f73b8fb60.png)